### PR TITLE
Styling/userview

### LINKED ISSUE #4: Setup Responsiveness

### LINKED ISSUE #18: As a USER: I should be able to see a “favorite” button on each card containing a displayed controversy. Once clicking this button, I should be able to see it from a “favorited” page on my personal dashboard. If no favorites are currently added, text reading “Hey! Add some favorites first!” will be in place of the button. The button can be clicked to view the favorited controversies page once the first favorited controversy is added.

### LINKED ISSUE #21: Add PropTypes

### LINKED ISSUE #23: Responsive Design

### LINKED ISSUE #24: Error bad routes && Error Handling

### PR DESCRIPTION
**Pull request by Jesuitman**
What I did:
added in styling for userview in the profile, also added in proptypes, thank you joshi. Cypress testing is present on main now, but should be ignored for the sake of testing on our [testing specific branch](https://github.com/Jesuitman/m3-H8rAid/pull/42).

Did this break anything?

- [x] No
- [ ]  Yes

Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x]  Styling 
- [ ]  Bug fix (non-breaking change which fixes an issue)
- [ ]  Refactor(DRY-ing up/ reorganizing code, etc.)
- [x]  Super small fix (Corrected a typo, removed a comment, etc.)
- [ ]  Skip all the other stuff and briefly explain the fix.

Checklist:

- [ ]  If this code needs to be tested, all tests are passing
- [x]  The code runs locally
- [ ]  There are comments where clarification/ organization is needed
- [x]  The code is DRY. If it's not, I tried my best
- [x]  I reviewed my code before pushing

What's next?
this is a finished product

### CODE REVIEW

**Review by tenthwalker — APPROVED** [review]
Thanks for helping me get this wrapped up Lex, you're the best!

**Comment by MaryBruff** [comment]
Beauty! Great work!

File: src/App/App.js [hunk 1]
```diff
@@ -44,38 +45,40 @@ function App() {
...
+              onSaveAsFavorite={() => saveControversy(item.parse.text["*"], true)}
+            />
+          ))}
           </section>
         </section>
-      )}
-    </div>
+        )}
+      </div>
+    </ErrorBoundary>
```
Comment (MaryBruff) [inline src/App/App.js:81]: Nice

File: src/UserView/UserView.css [hunk 2]
```diff
@@ -0,0 +1,17 @@
+.profile {
+    flex-direction: column;
+  }
+  
+  .filter-buttons, .profile {
+    display: flex;
```
Comment (MaryBruff) [inline src/UserView/UserView.css:6]: She cute

**Pull request merged**